java-manta-client/src/main/java/com/joyent/manta/util/CipherCloner.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016-2017, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.util;
+
+import com.joyent.manta.client.crypto.ExternalSecurityProviderLoader;
+import com.joyent.manta.exception.MantaMemoizationException;
+
+import javax.crypto.Cipher;
+import java.security.Provider;
+
+/**
+ * Utility class for cloning Cipher objects.
+ */
+public final class CipherCloner implements Cloner<Cipher> {
+
+    /**
+     * Instance of object-cloning library that does deep generic cloning.
+     */
+    private static final com.rits.cloning.Cloner INSTANCE = new com.rits.cloning.Cloner();
```

Comment:
The `Cloner<T>` interface exists in the same package so the symbol clashes. That  interface was originally an abstract class that behaved similarly to `AbstractSerializer` but the limited need for reflection meant it's less useful now. We could drop the mostly-useless interface or rename it (`Copier` springs to mind for some odd reason).